app/src/main/java/com/sebastianvm/mango/model/Taxes.kt

```diff
@@ -0,0 +1,97 @@
+package com.sebastianvm.mango.model
+
+import androidx.annotation.StringRes
+import com.sebastianvm.mango.R
+import com.sebastianvm.mango.database.models.TaxEntity
+import java.math.BigDecimal
+
+enum class Taxes(@StringRes val taxName: Int, val taxEntity: TaxEntity) {
```

Comment:
i think we should consider linking our source either in notion or source code, but preferably source code